net-p2p/bitcoin-qt/bitcoin-qt-22.0.ebuild

```diff
@@ -132,8 +132,13 @@ src_prepare() {
 		eapply "${knots_patchdir}/${KNOTS_P}_p5-ts.patch"
 	fi
 
+	eapply "${FILESDIR}/22.0-compat-libsecp256k1-0.1_pre20210628.patch"
+
 	eapply_user
 
+	# fix Bashism (see cf7292597e18ffca06b0fbf8bcd545aec387e380)
+	sed -e 's/==/=/g' -i configure.ac || die
```

Comment:
I don't really like sed for this. It could easily touch something else.

Reply:
I only used `sed` because it was the tiniest way to accomplish the goal. I can just as easily fetch a patch from upstream and apply it, but that's using a sledgehammer to push in a thumbtack. Will do.

Reply:
It's less about the change itself, and more that `sed` expressions tend to rot and someone ends up forgetting to clean them up and they either end up as clutter or being harmful. I understand though.